x/feegrant/client/testutil/suite.go

```diff
@@ -239,7 +239,7 @@ func (s *IntegrationTestSuite) TestCmdGetFeeGrants() {
 		tc := tc
 
 		s.Run(tc.name, func() {
-			cmd := cli.GetCmdQueryFeeGrants()
+			cmd := cli.GetCmdQueryFeeGrantsByGrantee()
 			out, err := clitestutil.ExecTestCLICmd(clientCtx, cmd, tc.args)
```

Comment:
Can you also add integration tests for `GetCmdQueryFeeGrantsByGranter`

Reply:
Done 👍